collect_app/src/androidTest/java/org/odk/collect/android/utilities/BitmapScaledToDisplayTest.java

```diff
@@ -17,94 +17,40 @@
 
     @Test
     public void scaleDownBitmapWhenPossible() {
-        new ScaleImageTest()
-                .createBitmap(1000, 1000)
-                .scaleBitmapToDisplay(500, 500)
-                .assertScaledBitmapDimensions(500, 500);
-
-        new ScaleImageTest()
-                .createBitmap(600, 800)
-                .scaleBitmapToDisplay(600, 200)
-                .assertScaledBitmapDimensions(150, 200);
-
-        new ScaleImageTest()
-                .createBitmap(500, 400)
-                .scaleBitmapToDisplay(250, 200)
-                .assertScaledBitmapDimensions(250, 200);
-
-        new ScaleImageTest()
-                .createBitmap(2000, 800)
-                .scaleBitmapToDisplay(300, 400)
-                .assertScaledBitmapDimensions(333, 133);
+        runScaleTest(1000,   1000,   500,    500,    500,    500,    false);
```

Comment:
Sorry to be picky, but the units positions and commas should line up, for maximum happiness.
```
runScaleTest( 600, ...
runScaleTest(2000, ...
```

Reply:
Does this look okay now?